.github/workflows/publish-release.yml

```
@@ -43,9 +43,16 @@ jobs:
           if [[ -z "${RELEASE_BODY}" && -f "CHANGELOG.md" ]]; then
             RELEASE_BODY="See [CHANGELOG.md](CHANGELOG.md) for changes details."
           fi
+          # Determine if the release is a prerelease
+          if [[ "${TAG_NAME}" =~ -(alpha|beta|rc)[0-9]*$ ]]; then
+            PRERELEASE=true
+          else
+            PRERELEASE=false
```

I cannot tell if every shell handles `true`/`false` as strings. Adding quotes may be safer.

```suggestion
            PRERELEASE="true"
          else
            PRERELEASE="false"
```